client.py

```diff
@@ -221,7 +270,7 @@ def games_were_tracking_string():
 def format_game_for_reply(game, game_id):
     url = f'steam://openurl/https://store.steampowered.com/app/{game_id}'
     name = game['name']
-    discounted_percent = game['price_overview']['discount_percent']
+    discounted_percent = game['price_overview']

```

Comment:
Maybe good time to rename discounted_percent since it is actually holding price_overview I believe

Reply:
I honestly am not sure why I changed this, I think this would actually break it since discounted_percent is later referenced in an f string, so I definitely just want it to have the number not the entire price_overview. I have reverted this.